### PR TITLE
"all" default selector (only for freq app)

### PR DESCRIPTION
**Pull request by amsan7**
- freq app on website is updated if you wanted to actually try it

- there's a transcript where the child is listed as 19 years old (author probably meant 19 months) which is the reason for the long range slider

### CODE REVIEW

File: freq/server.R [hunk 1]
```diff
@@ -147,19 +159,25 @@ server <- function(input, output, session) {
...
            aes(x = age_y,
                y = ppm,
                col = speaker_role)) +
       geom_point() +
       geom_smooth(se=FALSE, method = "loess", span=1) + 
-      facet_wrap(~target_child_name) +
       ylab("Frequency (parts per million words)") + 
       xlab("Target Child Age (years)") + 
       xlim(input$age_range[1], input$age_range[2]) +
+      scale_y_continuous(limits = c(0, 50000)) +
```
Comment (amsan7) [inline freq/server.R]: what's the best way to do this? plot very zoomed out when you include outliers

**Comment by amsan7** [comment]
also side note this repo isn't watched on slack (i can't add as admin perms required)

**Comment by mcfrank** [comment]
some thoughts on getting grouping to work:

## model 1
```
if (target child == "all") 

  blah %>%
    group_by(other stuff) %>%
    get freqs

else

  blah %>%
    group_by(CHILD, other stuff) %>%
    get freqs
```

## model 2
```
group_vars <- quos(corpus, age_bin, etc.)

group_var <- ifelse(target_child == "all", group_vars, 
                    append(group_vars, tatget_child)) 

blah %>%
  group_by(UQ(group_vars))
```


https://github.com/tidyverse/rlang
http://maraaverick.rbind.io/2017/08/tidyeval-resource-roundup/
https://edwinth.github.io/blog/dplyr-recipes/?utm_content=buffer51c64&utm_medium=social&utm_source=twitter.com&utm_campaign=buffer

**Comment by mcfrank** [comment]
in particular, goal is to get averages *across* units when "all" is selected, but *within* units when specific units are selected, e.g. "all" children gives you one mean, but child x and child y gives you separate series and separate panels for them.

**Pull request merged**